tests/fixtures/src/TokenModel.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * @author Chris Chabilall <chris.c@vanillaforums.com>
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license http://www.opensource.org/licenses/gpl-2.0.php GNU GPL v2
+ */
+
+namespace VanillaTests\Fixtures;
+
+use Vanilla\TokenSigningTrait;
+
+/**
+ * Class TokenModel.
+ * Used for TokenSigningTraitTest.
```

Comment:
If this is *only* for testing that trait, the model should probably have a name to reflect that. As it stands, its name implies it's a general-use class for testing tokens. I think the description should be updated to reflect it's a class for testing tokens *or* the class should be renamed to better indicate it's only for testing that one trait.